docs/example-snapshot.yaml

```
@@ -0,0 +1,7 @@
+everest-core:
+  git: git@github.com:EVerest/everest-core.git
+  git_rev: de44323b8040758b285b1ad2b0338bbe442ee214
+  git_tag: 2022-11.0
+everest:
+  git: git@github.com:EVerest/EVerest.git
+  git_tag: feature/improve-modularity
```

Before Merge:

```suggestion
  git_tag: main
```
or

```suggestion
  git_tag: 2022-11.0
```